app/controllers/items/search_location.js

```diff
@@ -105,16 +107,33 @@ export default searchModule.extend({
     movePartialQty(){
       var location = this.get("selectedLocation");
       var item = this.get("item");
-      var packagesLocationQty = localStorage['packagesLocationQty'];
-      var totalQty = localStorage["totalQty"];
+      var packagesLocationQty = [];
+      var totalQty = '';
+      if(this.get('skipScreenForSingletonItem')) {
+        var record = {};
+        var packages_location = this.get('store').peekRecord('packages_location', this.get('pkgsLocationId'));
+        record["packages_location_id"] = this.get('pkgsLocationId');
+        record["location_id"] = packages_location.get('locationId');
+        record["package_id"] = packages_location.get('packageId');
+        record["new_qty"] = "1";
```

Comment:
Instead of defaulting it to 1 can we take quantity of package? I know we are handling only single quantity items from stock app. But still I feel if we are moving item then we should send package quantity instead of assigning it 1.  Just suggestion.